### PR TITLE
(packaging) Prepare for 0.12.1 release

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by Magisus** [comment]
This commit cuts out the breaking change for Ruby 2.4, see https://github.com/Magisus/leatherman/commits/0.12.1-release.

**Comment by MikaelSmith** [comment]
Shouldn't this target something besides master?

**Comment by Magisus** [comment]
Hmm, not sure. 0.12.0 is not on stable though.

**Comment by MikaelSmith** [comment]
We need to create a new branch - puppetlabs/leatherman#0.12.1-release - and merge this to it. Then tag off that branch, and merge it back to puppetlabs/leatherman#master.

**Comment by Magisus** [comment]
That works too. It's pretty much equivalent to this, except the release branch was on my fork. So long as we have the SHA, we can tag it, and it knows what its parent is (I learned this when @branan did it for Puppet recently). But I'm happy to create the extra branch here too for clarity.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.12.1]
+
+### Fixed
+- Locale files are installed relative to the Leatherman install root, taking into account support for relocatable packages (LTH-135)
+
```
Comment (MikaelSmith) [inline CHANGELOG.md]: I'd note that the `Ruby API binding for rb_ll2inum` change from 0.11.2 is re-introduced to 0.12 here.

**Pull request closed without merging**